src/library-authoring/edit-block/LibraryBlock/wrap.js

```diff
@@ -376,7 +377,7 @@ export default function wrapBlockHtmlForIFrame(html, resources, lmsBaseUrl) {
     <!-- A Studio-served stylesheet will set the body min-height to 100% (a common strategy to allow for background
     images to fill the viewport), but this has the undesireable side-effect of causing an infinite loop via the
     onResize event listeners in certain situations.  Resetting it to the default "auto" skirts the problem. -->
-    <body style="min-height: auto">
+    <body style="min-height: auto; background: white; padding: 12px">
```

Comment:
Is it necessary to set the white color here? It might be better to use a `transparent` color here if it works.

Reply:
hm, seems to work!) changed